### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by tatellos**
I fixed a few things, most importantly a few fields had the wrong names or were missing.

### CODE REVIEW

File: frontend/components/RestaurantForm.vue [hunk 1]
```diff
@@ -84,6 +85,8 @@
           type="tel"
```
Comment (Empty2k12) [inline frontend/components/RestaurantForm.vue:85]: Code Smell?

File: frontend/components/RestaurantForm.vue [hunk 2]
```diff
@@ -126,18 +129,16 @@
...
         </b-form-tag>
       </div>
     </b-form-group>
-    <b-button v-if="addressEntered" type="submit" class="submit">
-      <b>{{ submitText }}</b>
-    </b-button>
-    <b-button v-else type="submit" class="submit">
-      Weiter
+    <b-button type="submit" class="submit wr-button main">
+      <span v-if="!addressEntered">Weiter</span>
```
Comment (Empty2k12) [inline frontend/components/RestaurantForm.vue:140]: `v-if="addressEntered"` ?

File: frontend/components/RestaurantForm.vue (hunk 1 above)
Comment (tatellos) [reply frontend/components/RestaurantForm.vue:85]: I didn't touch that line. Also, it's useful to get a numbers-keyboard in this field

File: frontend/components/RestaurantForm.vue (hunk 2 above)
Comment (tatellos) [reply frontend/components/RestaurantForm.vue]: I'll do v-if="addressEmpty" to keep the order of the code the same as the order in time

**Comment by wiomoc** [comment]
Merge conflicts present!

**Comment by tatellos** [comment]
Jo, jetzt wo edit restaurants gemerget ist, braucht man diesen branch nicht mehr. Das sind die gleichen Änderungen.

**Pull request closed without merging**